internal/common/common.go

```diff
@@ -1,5 +1,7 @@
 package common
 
+var Version string = "dev"
```

Comment:
Is this `cli` specific or the version of both projects, also the http client?
If both, then its fine. If its only for the CLI I think it make sense to rename it.. `CliVersion` 🤷

Reply:
Its for both otherwise I wouldn't put it in common package!